src/app/shared/ui-kit/ui-button/ui-button.component.ts

```diff
@@ -0,0 +1,16 @@
+import { CommonModule } from '@angular/common';
+import { Component, Input } from '@angular/core';
+
+@Component({
+  selector: 'quiz-ui-button',
+  standalone: true,
+  imports: [CommonModule],
+  templateUrl: './ui-button.component.html',
+})
+export class UiButtonComponent {
+  @Input() route: string = '';
+
+  onClick() {
+    alert(`opening catalog page ${this.route}`);
```

Comment:
Let's add an EventEmitter here to emit click event

Reply:
resolved